### PR TITLE
adding pointer to reports page

### PR DESCRIPTION
**Pull request by pdg744**
The reports.risczero.com page has info that will help users make sense of the info on `recursion.md`. 
This PR adds an `info` admonition to the top of `recursion.md` in order to help users find that information.

### CODE REVIEW

**Comment by pdg744** [comment]
@nahoc this is my first time trying to use the new linting setup, and i'm getting the following error: 

image

I guess I need to change my python configuration?

**Comment by nahoc** [comment]
> @nahoc this is my first time trying to use the new linting setup, and i'm getting the following error:
> 
> image
> I guess I need to change my python configuration?

tbh I couldn't get the python stuff to work either (I'm not the one who put that in) but changing it to `python3` worked for me locally... idk why but aliasing it didnt seem to work inside bash scripts

File: website/api/recursion.md [hunk 1]
```diff
@@ -143,5 +147,6 @@ The final step in the recursion process is `compress()`, which outputs a [Groth1
 [proof system]: /proof-system/proof-system-sequence-diagram
 [Prover]: https://docs.rs/risc0-zkvm/latest/risc0_zkvm/trait.Prover.html#method.prove_with_opts
 [receipt claim]: /terminology#receipt-claim
+[reports.risczero.com]: https://reports.risczero.com
```
Comment (SchmErik) [inline website/api/recursion.md:150]: you need to also add this to the other copy of recursion.md under version-1.0

**Review by SchmErik — APPROVED** [review]
LGTM

**Pull request merged**